roles/openshift-defaults/defaults/main.yml

```diff
@@ -6,4 +6,12 @@ openshift_password: ""
 openshift_project_name: ""
 openshift:
   common:
-    client_binary: "oc"
+    client_binary: "oc"  
+docker:
+  common:
+    client_binary: "docker"
+  registry:
+    host: registry.env3-1.innovation.labs.redhat.com
```

Comment:
Just a default. Can be overridden at execution time so there is no need to rely on whats in the source. We could eliminate the default entirely and rely on an environment variable or the like.